build-system/compile/single-pass.js

```diff
@@ -324,7 +322,18 @@ exports.getGraph = function(entryModules, config) {
     // directly and which we don't want to apply during deps finding.
     .transform(babelify, {
       compact: false,
-      plugins: ['transform-es2015-modules-commonjs'],
+      plugins: [
+      'transform-es2015-modules-commonjs',
+      '@babel/plugin-transform-react-constant-elements',
```

Comment:
@jridgewell @rsimha is there a place we can unify the declaration of these new plugins? (it needs to be used here and in babel.config.js)

Reply:
What if you remove the entire settings object from here? Won't `babelify` default to using whatever is in `babel.config.js`?

In other words, change this code...

https://github.com/ampproject/amphtml/blob/79e2715683d905c5809c6dea1de64befd68beb6e/build-system/compile/single-pass.js#L315-L328

... to:

```js
  // Use browserify with babel to learn about deps.
  const b = browserify(entryModules, {
    debug: true,
    deps: true,
    detectGlobals: false,
    fast: true,
    browserField: 'module',
  })
    // The second stage are transforms that closure compiler supports
    // directly and which we don't want to apply during deps finding.
    .transform(babelify);
```

Reply:
If this doesn't work, babel already performs merging which might work: https://babeljs.io/docs/en/options#merging.

If not, require the `babel.conf.js` file, and modify the config here.